src/values/ECMAScriptFunctionValue.js

```diff
@@ -23,4 +24,6 @@ export default class ECMAScriptFunctionValue extends FunctionValue {
   $ThisMode: "lexical" | "strict" | "global";
   $HomeObject: void | ObjectValue;
   $FunctionKind: "normal" | "classConstructor" | "generator";
+  activeArguments: void | Map<BabelNodeSourceLocation, [number, Array<Value>]>;
```

Comment:
I am a bit unclear what this represents. The current (possibly abstract) arguments for a given call site?

Reply:
I'll add a comment. This is the place where we keep track of the arguments of the most recent active call to the function from a particular location. It serves as an optimization since without it we'd have to traverse the stack upon every call in order to detect recursive calls.